var/spack/repos/builtin/packages/py-tweedledum/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyTweedledum(PythonPackage):
+    """tweedledum is a library for synthesis, compilation, and
+    optimization of quantum circuits. The library is written to be
+    scalable up to problem sizes in which quantum circuits outperform
+    classical ones. Also, it is meant to be used both independently
+    and alongside established tools."""
+
+    homepage = "https://github.com/boschmitt/tweedledum"
+    pypi = 'tweedledum/tweedledum-1.1.1.tar.gz'
+
+    version('1.1.1', sha256='58d6f7a988b10c31be3faa1faf3e58288ef7e8159584bfa6ded45742f390309f')
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools@42:', type='build')
+    depends_on('py-scikit-build@0.12:', type='build')
+    depends_on('cmake@3.18:', type='build')
+    depends_on('ninja', type='build')
```

Comment:
Python build systems only understand Python libraries, so I think they actually mean `py-cmake` and `py-ninja`

Reply:
hm, they don't seem to actually be using that directly, but switching to what it explicitly states isn't a bad idea.

Reply:
Yeah, I think it's just the Pythonic way of ensuring that those non-Python deps are installed. If `py-cmake` is installed correctly, then presumably `cmake` is too. The only reason we should care is that `pip` and `setuptools` will install those deps for us if we don't install them ourselves.